storage_scrubber/src/main.rs

```diff
@@ -208,21 +208,21 @@ async fn main() -> anyhow::Result<()> {
                         }
 
                         if summary.is_fatal() {
-                            Err(anyhow::anyhow!("Fatal scrub errors detected"))
+                            eprintln!("Fatal scrub errors detected");
```

Comment:
should be an `error!` log line?

Reply:
6b664176848781120da6a4951a7b183c3e73114d